incubator/jaeger/values.yaml

```diff
@@ -40,6 +40,9 @@ storage:
     user: user
     usePassword: true
     password: password
+    tls:
```

Comment:
this value should not be in between `password` and `existingSecret`... makes things confusing

Reply:
Right.